Unix/etc/omiserver.conf

```diff
@@ -16,51 +16,36 @@
 #idletimeout=TIMEOUT
 
 ##
-## trace -- enable tracing to standard output (default is 'false')
-##
-#trace=(true|false)
-
-##
-## loglevel -- set the log level of the server
+## loglevel -- set the loggiing options for MI server
+##   Valid options are: ERROR, WARNING, INFO, DEBUG, VERBOSE (debug builds only)
+##   If commented out, then default value is: WARNING
 ##
 loglevel = WARNING
```

Comment:
Comment this out please. We don't want to force the default, we want the default to be commented out.